lib/pf/Authentication/Source/OpenIDSource.pm

```diff
@@ -0,0 +1,71 @@
+package pf::Authentication::Source::OpenIDSource;
+
+=head1 NAME
+
+pf::Authentication::Source::OpenIDSource
+
+=head1 DESCRIPTION
+
+This module implements methods for the OpenID source and the methods necessary
+to perform the OAuth flow since Net::OAuth2 lacks support for OpenID OAuth.
```

Comment:
Hum, that's not true, seems like a copy/paste from the Twitter source but this doesn't apply.
Use a description similar to Facebook, Google, Github, etc